crates/blockchain/api/src/lib.rs

```diff
@@ -135,6 +146,8 @@ pub trait ReserveBlocks {
 
     /// Reserves the provided number of blocks, starting from the next block
     /// number.
+    // TODO: analyze if we can receibe the BlockConfig here so blockachain does not
+    // have to keep track of it
```

Comment:
Suggestion: I created an issue so we can keep track of this TODO:

```suggestion
    // TODO: https://github.com/NomicFoundation/edr/issues/1228
    // Analyze whether we can receive the BlockConfig here so blockachain does not
    // have to keep track of it
```